Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -1430,7 +1464,6 @@ namespace O3DE::ProjectManager
             try
             {
                 // required
-                gemRepoInfo.m_repoUri = Py_To_String(data["repo_uri"]);
```

Comment:
Yes, this was to use the URI as entered and used in the manifest since that is the one that is also used to generate the hash for storage in the cache folder.